nncf/torch/layers.py

```diff
@@ -156,6 +156,19 @@ def from_module(module):
         return nncf_bn
 
 
+class NNCFInstanceNorm(_NNCFModuleMixin, nn.InstanceNorm3d):
```

Comment:
Why only nn.InstanceNorm3d?

Reply:
The target of this PR was to support a specific model(with only `nn.InstanceNorm3d`) from the ticket.
But i see no problem for adding `nn.InstanceNorm1d`, `nn.InstanceNorm2d`. It is also strange that we only wrap `nn.BatchNorm2d`)

Reply:
At the very least you should call the wrapper NNCFInstanceNorm**3d**.